t/app/controller/admin/users.t

```diff
@@ -685,7 +685,9 @@ subtest "Removing account from admin" => sub {
     $user->discard_changes;
     is $user->name, '', 'Name gone';
     is $user->password, '', 'Password gone';
-    is $user->email, 'removed-' . $user->id . '@example.org', 'Email gone'
+    is $user->email, 'removed-' . $user->id . '@example.org', 'Email gone';
+    $mech->get_ok( '/admin/report_edit/' . $user->problems->first->id );
+    $mech->content_contains("name='username' value=''>")
```

Comment:
Missing the trailing semicolon here - might not matter syntactically but saves a future headache if it's not spotted.

Reply:
👍 WEIRD how I sort of copied that from what it was before!